configs/components/fesom/fesom.yaml

```diff
@@ -359,13 +466,8 @@ coupling_fields:
                 grid: feom
         sie_feom:
                 grid: feom
-        snt_feom:
+        snt_feom: # snow temp
                 grid: feom
-                #description: |
-                #       whatever snt means
-                #unit: kg m s
-                #standard_cf_name: 'shit no tuna?'
-                #cf_code: 42
```

Comment:
Oh no! The epic "shit no tuna" is gone :'( , one of my first memories of reading the yaml files for the first time

Reply:
Yeah sorry. I thought its time to move on ;)